tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html

```diff
@@ -1071,14 +1074,17 @@ <h2>Continue...</h2>
         this.$$('#left-menu').style.width = leftMenuWidth + 'px';
 
         const windowWidth = this._getWindowWidth();
+        // We add 8px of padding to the main content.
         this.$$('#center-content').style.width = (
-            windowWidth - leftMenuWidth - this._resizerWidth) + 'px';
+            windowWidth - leftMenuWidth - this._resizerWidth - 8) + 'px';
```

Comment:
Maybe add a comment to explain this magic number? Or even better: give this magic number a name.